Team9Project/src/com/indragie/cmput301as1/ExpenseClaimDetailActivity.java

```diff
@@ -225,7 +282,56 @@ public boolean onCreateOptionsMenu(Menu menu) {
 	@Override
 	public boolean onPrepareOptionsMenu(Menu menu) {
 		super.onPrepareOptionsMenu(menu);
-		menu.findItem(R.id.action_add_item).setEnabled(editable);
+		boolean UserCheck = user.getName().contentEquals(claim.getUser().getName());//SHOULD BE ID USING NAME FOR TESTING
+
+		if (status ==Status.APPROVED){
+			menu.findItem(R.id.action_add_item).setEnabled(false);
+			menu.findItem(R.id.action_mark_submitted).setEnabled(false);
+			menu.findItem(R.id.action_mark_approved).setEnabled(false);
+			menu.findItem(R.id.action_mark_returned).setEnabled(false);
+		}
+		if(status == Status.RETURNED){
+			if(UserCheck){
+				menu.findItem(R.id.action_add_item).setEnabled(true);
+				menu.findItem(R.id.action_mark_submitted).setEnabled(true);
+				menu.findItem(R.id.action_mark_approved).setEnabled(false);
+				menu.findItem(R.id.action_mark_returned).setEnabled(false);
```

Comment:
perhaps since some stuff like "menu.findItem(R.id.action_mark_approved).setEnabled(false);" is found in both places of the if and else statement, it might be easier to if just appears once?